cmd/single-ocsp/main.go

```diff
@@ -26,7 +26,7 @@ type PKCS11Config struct {
 	TokenLabel      string
 	PrivateKeyLabel string
 	PIN             string
-	SlotID          float64
+	SlotID          uint
```

Comment:
Note that that article is about C/C++.  In Go, with the libraries we're using and how we're using these things, we actually [get more safety from `uint`](http://play.golang.org/p/R5Mwk9hZfZ).

* `encoding/json` will check that the input value is compatible with `uint` and error if not
* Go will prevent negative constants or signed values from being put into `uint` arguments

The only risk remaining is that you underflow (which Go still allows), but since no arithmetic is going on with the slot ID (it's just getting passed around), this is not an issue.